src/dark/PortfolioLanding.jsx

```diff
@@ -64,6 +64,14 @@ const PortfolioLanding = () => {
                 </div>
             </div>
             {/* End Slider Area   */} 
+            
+                {/* Start Service Area  */}
+                <div className="service-area ptb--120 bg_color--5">
+                   <div className="container">
+                        <ServiceList />
+                   </div>
+                </div>
+                {/* End Service Area  */}

```

Comment:
Hi @LiviaFResende,
It looks like the Services section is broken. You're not in trouble, but please make sure to check your changes on your local machine before sending in a PR.

![image](https://user-images.githubusercontent.com/8443215/200461412-8336937f-89fd-466f-a9fe-240fb93cbb1b.png)

Can you please copy and paste the section code from this file? The red lines are the lines that were removed.

https://github.com/codesydney/lloydconsulting/commit/09e054d405de592c7f458b162329b4c156f8ca85#diff-be3c8e4a245b8ba5180f3063a19dfce031d1c49760e7008d64660de11da90ac6

Reply:
I'll fix this one for you. You don't have to worry about this anymore.